app/tests/spec/lib/xss.js

```diff
@@ -39,7 +39,9 @@ function (chai, _, XSS, Constants) {
       });
 
       it('disallows javascript: href', function () {
-        expectEmpty('javascript:alert(1)'); // jshint ignore:line
+        /*eslint-disable */
```

Comment:
That actually didn't work, it made either the jshint tests fail or the eslint tests fail. So I went for the block disable one.